cmd/health-monitor_test.go

```diff
@@ -68,6 +71,19 @@ type hmonTestSuite struct {
 	assert *assert.Assertions
 }
 
+func generateRandomPID() string {
+	rand.Seed(time.Now().UnixNano())
+	var randpid int
+	for i := 0; i <= 5; i++ {
+		randpid = rand.Intn(9000) + 1000
```

Comment:
Make the random pid length as 5

```suggestion
		randpid = rand.Intn(90000) + 10000
```